rudder-webapp/SOURCES/Makefile

```diff
@@ -188,7 +187,6 @@ install: build initial-promises initial-ncf
 	install -m 640 slapd.conf $(DESTDIR)/opt/rudder/etc/openldap/slapd.conf
 	install -m 644 $(WEBAPPDIR)/ldap-inventory/inventory-repository/src/main/resources/ldap/inventory.schema $(DESTDIR)/opt/rudder/etc/openldap/schema/
 	install -m 644 $(CORE_RESOURCES)/ldap/rudder.schema $(DESTDIR)/opt/rudder/etc/openldap/schema/
-	install -m 644 rsyslog/rudder-slapd.conf $(DESTDIR)/etc/rsyslog.d/rudder-slapd.conf
```

Comment:
don't we need to also remove that at postinst time ?

Reply:
This file is for slapd logs and not linked to reporting

Reply:
(it is not removed anymore in this PR)